packages/mobile/src/screens/chat-screen/UserChatHeader.tsx

```diff
@@ -1,41 +1,23 @@
 import type { User } from '@audius/common/models'
-import { TouchableOpacity } from 'react-native'
+import { css } from '@emotion/native'
 
+import { Flex } from '@audius/harmony-native'
 import { ProfilePicture } from 'app/components/core'
-import UserBadges from 'app/components/user-badges'
-import { useNavigation } from 'app/hooks/useNavigation'
-import { makeStyles } from 'app/styles'
-
-import type { AppTabScreenParamList } from '../app-screen'
-
-const useStyles = makeStyles(({ spacing, palette, typography }) => ({
-  profileTitle: {
-    display: 'flex',
-    flexDirection: 'row',
-    alignItems: 'center'
-  },
-  userBadgeTitle: {
-    fontSize: typography.fontSize.medium,
-    fontFamily: typography.fontByWeight.bold,
-    color: palette.neutral
-  }
-}))
+import { UserLink } from 'app/components/user-link'
 
 export const UserChatHeader = ({ user }: { user: User }) => {
-  const styles = useStyles()
-  const navigation = useNavigation<AppTabScreenParamList>()
   return (
-    <TouchableOpacity
-      onPress={() => navigation.push('Profile', { id: user.user_id })}
-      style={styles.profileTitle}
+    <Flex
+      direction='row'
+      justifyContent='center'
+      alignItems='center'
+      style={css({
+        maxWidth: '70%'
+      })}
+      gap='s'
     >
-      <ProfilePicture
-        userId={user.user_id}
-        size='small'
-        mr='s'
-        strokeWidth='thin'
-      />
-      <UserBadges user={user} nameStyle={styles.userBadgeTitle} />
-    </TouchableOpacity>
+      <ProfilePicture userId={user.user_id} size='small' strokeWidth='thin' />
```

Comment:
does this go to the profile on touch? wondering if that matters

Reply:
the profile pic does not.. cc @dylanjeffers maybe we want to add this to UserLink.
i can re-wrap in the touchable opacity again.

Reply:
done